molo/yourtips/migrations/0002_create_your_tips_index_pages.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def create_yourtips_index(apps, schema_editor):
+    from molo.core.models import Main
+    from molo.yourtips.models import YourTipsIndexPage
+    main = Main.objects.all().first()
+
+    if main:
+        tip_index = YourTipsIndexPage(title='Your Tips', slug='Your-tips')
```

Comment:
You use `Your tips` every where else instead of `Your Tips`. Maybe use that here also